proto/connector_service.proto

```diff
@@ -27,7 +27,6 @@ message SinkParam {
   string db_name = 5;
   string sink_from_name = 6;
   catalog.SinkFormatDesc format_desc = 7;
-  optional uint32 target_table = 8;
```

Comment:
The `SinkParam` is only used in rpc. So the proto breaking change can be ignored.